elcid/models.py

```diff
@@ -758,6 +782,47 @@ def get_api_name(cls):
         return camelcase_to_underscore(cls._meta.object_name)
 
 
+class ChronicAntifungal(omodels.PatientSubrecord):
```

Comment:
Is this definitely a patientsubrecord? e.g. should it also have it's own API endopint, and be serialised on every episode/patient context?